Sources/Swift/Integrations/UserFeedback/SentryUserFeedbackForm.swift

```diff
@@ -130,12 +130,19 @@ class SentryUserFeedbackForm: UIViewController {
             let list = missing.count == 1 ? missing[0] : missing[0 ..< missing.count - 1].joined(separator: ", ") + " and " + missing[missing.count - 1]
             let alert = UIAlertController(title: "Error", message: "You must provide all required information. Please check the following field\(missing.count > 1 ? "s" : ""): \(list).", preferredStyle: .alert)
             alert.addAction(UIAlertAction(title: "OK", style: .default))
-            present(alert, animated: config.animations)
+            present(alert, animated: config.animations) {
+                if let block = self.config.onSubmitError {
+                    block(NSError(domain: "io.sentry.error", code: 1, userInfo: ["missing_fields": missing, NSLocalizedDescriptionKey: "The user did not complete the feedback form."]))
```

Comment:
`m`: Wouldn't be [Swift errors](https://developer.apple.com/documentation/swift/error) be more suitable than an NSError? Basically using an enum or a struct.

Reply:
It would be great to add this info as a comment, cause I see myself asking the question again in 2 months when I look at this code again 😅.